tools/utils/Utils/AppxPackaging/AppxBundleMetadata.cs

```diff
@@ -57,8 +57,14 @@ public AppxBundleMetadata(AppxBundleMetadata parentBundleInfo, string fileRelati
         /// <summary>
         /// Gets the list of all the internal msix packages bundled in this bundle
         /// </summary>
+        [Obsolete]
         public IList<string> InternalAppxPackagesRelativePaths { get; private set; } = new List<string>();
 
+        /// <summary>
+        /// Gets the list of all the internal msix packages bundled in this bundle
+        /// </summary>
+        public IList<ChildPackageMetadata> ChildAppxPackages { get; private set; } = new List<ChildPackageMetadata>();

```

Comment:
Is there a reason to use `Child` instead of `Internal`? Just wondering as this was using Internal/External, but I couldn't find that distinction in the Win32 API. I'm okay with changing it if this is a more appropriate term.

Reply:
I was influenced by the parent method (PopulateChildAppxPackages), and I also thought Child was better than Internal. To me "Internal" has a connotation of internal code workings, like the internal keyword in c#, and I thought child was more descriptive. Totally fine to change it if you prefer otherwise!

Reply:
That makes sense. If you think "Child" is better than "Internal" let's keep it that way.
Although an external package reference may also be considered a child, hopefully that is not confusing.